srcs/back/src/friends/friends.controler.ts

```diff
@@ -0,0 +1,33 @@
+import { Body, Controller, Get, Post, UseGuards } from "@nestjs/common";
+import { AuthGuard } from "@nestjs/passport";
+import { FriendsService } from "./friends.service";
+import { JwtPayload } from "src/auth/jwt/jwt-auth.strategy";
+import { UserDecorator } from "src/users/decorator/users.decorator";
```

Comment:
```suggestion
import { JwtPayload } from "../auth/jwt/jwt-auth.strategy";
import { UserDecorator } from "../users/decorator/users.decorator";
```
tjrs les imports relatif

Reply:
Ca marche merci. Je ne savais pas que c'etait une norme.